tcms/testplans/tests/test_views.py

```diff
@@ -9,6 +9,119 @@
                                   ProductFactory, VersionFactory, PlanTypeFactory)
 
 
+class NewPlanViewTest(LoggedInTestCase):
```

Comment:
There are some improvements here but generally looks like a copy of `NewPlanViewTest` from `testplans/tests/test_new_plan.py`.  Add your changes (+ the change of the base class) to `test_new_plan.py` so we can review again.

Reply:
Roger that, I mostly used the changed from the `EditPlanViewTest` didn't know there already was an implementation at `test_new_plan.py`.